### ISSUE TITLE
達成方法集 G71の翻訳

### ISSUE THREAD

**Issue by momdo** [waic/wcag21#390]
対象ファイル：
https://github.com/waic/wcag21/blob/techniques-20191001/techniques/general/G71.html

Techniques for WCAG 2.0にも同名のファイルがある文書です。

**分割ワークフローを試験的に導入しています**
作業の進め方は：Techniques for WCAG 2.1の作業の進め方（分割ワークフロー版）を参照ください。
https://github.com/waic/wcag21/blob/techniques-20191001/work-step_split.md

**Comment by yaytom-ba** [comment]
置換作業完了しました。
ご確認お願いします。
差分3箇所になります。

■達成方法に関する重要な情報
```diff
- See Understanding Techniques for WCAG Success Criteria for important information about the usage of these informative techniques and how they relate to the normative WCAG 2.0 success criteria. The Applicability section explains the scope of the technique, and the presence of techniques for a specific technology does not imply that the technology can be used in all situations to create content that meets WCAG 2.0.
+ See Understanding Techniques for WCAG Success Criteria for important information about the usage of these informative techniques and how they relate to the normative WCAG 2.1 success criteria. The Applicability section explains the scope of the technique, and the presence of techniques for a specific technology does not imply that the technology can be used in all situations to create content that meets WCAG 2.1.
```
※リンク先も変更
2.0か2.1の差分でしたので、以下の内容で翻訳しました。
問題ないか合わせて確認おねがいします。
696a3cae15b6a74978411c61b5063f8f794739c6

■適用 (対象)
```diff
- Success Criterion 3.3.5 (Help)
- How to Meet 3.3.5 (Help)
- Understanding Success Criterion 3.3.5 (Help)
+ This technique relates to Success Criterion 3.3.5: Help (Sufficient).
```

■検証＞期待される結果の下の注釈
```diff
- If this is a sufficient technique for a success criterion, failing this test procedure does not necessarily mean that the success criterion has not been satisfied in some other way, only that this technique has not been successfully implemented and can not be used to claim conformance.
+ 
```
2.1では掲載されてないようでした。

**Comment by momdo** [comment]
@yaytom-ba 
ありがとうございます。問題なさそうです。

**Comment by momdo** [comment]
マージしたのでCloseします。
